runners/dmn-tck-runner-drools/src/test/java/org/omg/dmn/tck/runner/drools/DroolsTCKTest.java

```diff
@@ -516,7 +320,7 @@ private Object parseType(ValueType value, DMNType dmnType) {
                                 } catch (DateTimeParseException e) {
                                     return ComparablePeriod.parse(dateTimeOrDurationValue.toString());
                                 }
-                            } else if (dateTimeOrDurationValue instanceof XMLGregorianCalendar) {
```

Comment:
Does this change the behavior, somehow ? 
Why the ` if (dateTimeOrDurationValue instanceof XMLGregorianCalendar)` is not needed anymore ?

Reply:
The instanceof check is not needed, as there is this or here https://github.com/dmn-tck/tck/pull/677/files#diff-9b2a1d8823619346c392f6da61429acdbb6a4fbf31673bcba671639a492fa564R310